schemas/mozilla_nimbus_schemas/experiments/experiments.py

```diff
@@ -72,3 +72,4 @@ class NimbusExperiment(BaseModel):
     featureValidationOptOut: Optional[bool]
     localizations: Optional[dict]
     locales: Optional[List[str]]
+    publishedDate: Optional[str]
```

Comment:
Should this be a datetime object instead of string? Pydantic will accept and parse it for you if it's a string in ISO format. https://docs.pydantic.dev/1.10/usage/types/#datetime-types

Reply:
The other dates in here use string as well, I was mostly going off of that

Reply:
Works for me! Not worth changing them all for this PR so we can address it separately if we want.